sunpy/map/mapbase.py

```diff
@@ -195,21 +199,21 @@ def __init__(self, data, header, plot_settings=None, **kwargs):
         self._validate_meta()
         self._shift = SpatialPair(0 * u.arcsec, 0 * u.arcsec)
 
-        if self.dtype == np.uint8:
-            norm = None
-        else:
-            # Put import here to reduce sunpy.map import time
-            from matplotlib import colors
-            norm = colors.Normalize()
-
         # Visualization attributes
-        self.plot_settings = {'cmap': 'gray',
-                              'norm': norm,
-                              'interpolation': 'nearest',
-                              'origin': 'lower'
-                              }
+        self.cmap = cmap
+        self.norm = norm
         if plot_settings:
-            self.plot_settings.update(plot_settings)
+            self.plot_settings = plot_settings
+            warnings.warn("Handling of ``plot_settings`` is deprecated."
+                          "Subsequently setting ``plot_settings`` will have no effect on the plot."
+                          "Pass the plot specific settings to either ``.peek()`` or ``.show()``",
+                          DeprecationWarning)
+            if 'norm' in self.plot_settings:
+                self.norm = self.plot_settings['norm']
+            if 'cmap' in self.plot_settings:
+                self.cmap = self.plot_settings['cmap']
+        else:
+            self.plot_settings = {}
```

Comment:
What is the reason for an empty plot settings?

Reply:
> As discussed before, I think it is best to leave all the `cmap` and `norm` logic in the setters. However, I'll backtrack a bit and say that we should abandon the idea of using a property-setter approach for plot settings. Instead, I think the constructor should just look like
> 
> ```python
> def __init__(self, data, header, cmap='gray', norm=None, plot_settings=None, **kwargs):
>     # Other stuff
>     self.cmap = cmap
>     self.norm = norm
>     if plot_settings:
>         self.plot_settings = plot_settings
>         warnings.warn("Handling of plot_settings is deprecated. "
>                                  "Pass the plot specific settings to either peek() or show()",
>                                  DeprecationWarning)
>         if 'norm' in self.plot_settings:
>             self.norm = self.plot_settings['norm']
>         if 'cmap' in self.plot_settings:
>             self.cmap = self.plot_settings['cmap']
>     else:
>         self._plot_settings = {}
>     # Other stuff
> ```

@nabobalis As previously discussed in this thread and in the chatroom, ensuring 100% cross-compatibility is not possible. BY setting `plot_settings={}` we can ensure the plot settings other than `cmap` and `norm` are correctly handled by the plotting functions. Otherwise, `plot_settings` would default to `None` and one can't pass plot specific settings according to previous method.

Reply:
That's fair.

Reply:
Now that I see this, there's really no reason not to just leave it as `None`. Since we're not using the setter anymore, setting `plot_settings` (or any keys in it) will have no effect on how the Map is plotted.

Reply:
@wtbarnes so `plot_settings = {}` should be done or not?

Reply:
Yeah I don't really like the idea of an empty `dict` hanging around if we haven't set it. See my comments below. I don't think removing it affects anything else. Of course, this will break someone's code where they set `plot_settings` after the fact, but it won't have any affect anyway so 🤷‍♀